examples/js-console/src/main/webapp/index.html

```diff
@@ -20,6 +20,9 @@
     <button onclick="output(keycloak.createLoginUrl())">Show Login URL</button>
     <button onclick="output(keycloak.createLogoutUrl())">Show Logout URL</button>
     <button onclick="output(keycloak.createRegisterUrl())">Show Register URL</button>
+    <p>
```

Comment:
To make it consistent with the other features of the js-console just add button "Introspect Token" and put the result in 'output' instead of adding separate textarea